### ISSUE TITLE
Issue running RNAkinet

### ISSUE THREAD

**Issue by mfurla** [maragkakislab/rnakinet#1]
Dear developers,

I'm encountering some issues with the installation of RNAkinet within a dedicated Conda Environment.

The output of the pip install RNAkinet command appears to be fine, except for the following warning message:

WARNING: There was an error checking the latest version of pip.

However, when I execute rnakinet-inference --path ~/FAST5Tmp --output predictions_name.csv, I encounter several error messages associated with CUDA:

CUDA False
...
connector.py", line 549, in _choose_gpu_accelerator_backend
    raise MisconfigurationException("No supported GPU backend found!")
lightning_fabric.utilities.exceptions.MisconfigurationException: No supported GPU backend found!

Is there any chance you could assist me in resolving this issue? Additionally, do you happen to have a Docker image available that I could try running on my platform?

Thank you in advance for your help!

Best regards,
Mattia

P.S. I'm working with a Tesla P100 GPU.

**Comment by MartinekV** [comment]
Hi, thanks for letting us know about your issue!

We pushed a new version. Try to upgrade to this latest version of rnakinet (0.1.22).
If that doesn't fix the issue, could you please provide:

a) The system you are running this on (windows/ubuntu...)
b) The output of running `conda list` command
c) The output of running `nvidia-smi` command

Thank you
Vlasta

**Comment by mfurla** [comment]
Dear Vlasta,

thank you for your quick and kind response.

I updated RNAkinet. Now, I get the following (error) message running rnakinet-inference:

CUDA True
...
RuntimeError: Attempting to deserialize object on CUDA device 0 but torch.cuda.device_count() is 0. Please use torch.load with map_location to map your storages to an existing device.

The infos you required:

a) Rocky Linux 8.7
b) condaList.txt
c) nvidia-smi.txt

Thank you again,

Mattia
[nvidia-smi.txt](https://github.com/maragkakislab/rnakinet/files/13646367/nvidia-smi.txt)
[condaList.txt](https://github.com/maragkakislab/rnakinet/files/13646368/condaList.txt)

**Comment by MartinekV** [comment]
Hi Mattia,

Thanks for providing the files. We have tried to reproduce the issue on our multi-gpu machines, but we can't quite do it. Its possible that this is an issue with an older pytorch version failing to detect your GPU. We have pushed a new version (0.1.23) which updates the pytorch dependency to >=2.0 and may fix the error. Note this now requires python >=3.8.

Make sure you install this new version of rnakinet in a fresh conda environment. We have observed some problems with updating the package with pip, where other dependencies dont get updated correctly.

Let us know if it helped your issue, we're happy to help you get it running and improve the package in the meantime.
Best
Vlasta
